scene/gui/text_edit.cpp

```diff
@@ -3742,132 +3733,132 @@ void TextEdit::paste_primary_clipboard(int p_caret) {
 
 // Context menu.
 PopupMenu *TextEdit::get_menu() const {
-	if (!menu) {
+	if (!context_menu.is_initialized()) {
 		const_cast<TextEdit *>(this)->_generate_context_menu();
 	}
-	return menu;
+	return context_menu.get_main_menu();
 }
 
 bool TextEdit::is_menu_visible() const {
-	return menu && menu->is_visible();
+	return context_menu.is_visible();
 }
 
 void TextEdit::menu_option(int p_option) {
 	switch (p_option) {
-		case MENU_CUT: {
+		case TextEditMenu::MENU_CUT: {
```

Comment:
```suggestion
		case MENU_CUT: {
```
For all these, to keep it clean

Reply:
Done.